assets/css/style.css

```diff
@@ -415,8 +413,216 @@ footer a {
+    display: none;
+  }
+
+  .navMenu {
+    display: flex;
+    align-items: center;
+  }
+
+  .navItems {
+    list-style: none;
+    display: flex;
+  }
+
+  .navListItem {
+    padding: 4%;
+  }
+
+  .navItems li a {
+    text-decoration: none;
+  }
+
+  .headline {
+    background-image: none;
+    display: flex;
+    justify-content: center;
+    justify-items: center;
+    align-items: center;
+  }
+
+  .headline h1 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 4.8vw;
+    line-height: 125%;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin-top: 5%;
+    margin-left: 0;
+  }
+
+  .greenTxt {
+    margin: 0;
+  }
+
+  p {
+    font-size: 2vw;
+    margin: 0;
+    width: 60%;
+  }
+
+  .socialMedia {
+    display: flex;
+    margin-block: 0;
+  }
+
+  .moreWorks a img {
+    display: flex;
+    width: 40px;
+    height: 40px;
+    left: calc(50% - 40px / 2 + 0.5px);
+    bottom: calc(-38% - 40px / 2 + 0.5px);
+    position: relative;
+  }
+
+  .moreWorks h2 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 4vw;
+    line-height: 5.2vw;
+    margin-top: 0;
+    padding-top: 0;
+    text-align: center;
+    position: relative;
+    bottom: calc(-115% - 40px / 2 + 0.5px);
+  }
+
+  .works {
+    width: 100%;
+    display: grid;
+    grid-template-columns: auto auto auto;
+    padding: 10px;
+  }
+
+  .workSkills {
+    width: 100%;
+  }
+
+  .SkillsBTNs {
+    width: 100%;
+    height: auto;
+  }
+
+  .SkillsBTNs li button {
+    display: flex;
+    flex-direction: row;
+    background: #fff;
+    border-radius: 8px;
+    color: #36b37f;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    font-weight: 600;
+    margin-top: 13px;
+    border: 1px;
+    text-align: center;
+  }
+
+  .aboutMe {
+    background-image: url("/assets/images/about-me-bg-left-desktop.svg"), url("/assets/images/about\ me-right-\ desktop.svg");
+    background-repeat: no-repeat, no-repeat;
+    background-size: auto, auto;
+  }
+
+  .aboutMe p {
+    margin-bottom: 2%;
+  }
+
+  .form-section {
+    display: flex;
+    background-image: url("/assets/images/contact\ form-left-desktop.svg"), url("/assets/images/contact-form-bg-middle-desktop.svg"), url("/assets/images/contact-form-right-desktop.svg");
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position-x: right, center, left;
+    background-position-y: bottom, bottom, bottom;
+    background-size: 7%, 4%, 10%;
+  }
+
+  .form-section h2 {
+    font-size: 3vw;
+    align-items: center;
+    color: #172b4d;
+    font-weight: 700;
+  }
+
+  .formSection {
+    align-items: flex-start;
+  }
+
+  footer {
+    border-top: 1px solid #dfe1e6;
+    display: flex;
+    justify-content: center;
+    justify-items: center;
+    flex-direction: column;
+  }
+
+  footer ul {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
- [ ] Kindly make sure to align all social media links correctly to look like the provided design. :wink: 

  - ### Your design
 
   ![image](https://user-images.githubusercontent.com/87197412/186726691-d0e24399-570e-4e2f-adaa-7ad8447e6efb.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186726572-6feffd66-c256-44a5-86f4-dfba76e0f938.png)

Reply:
update the  social media space